src/views/Order.vue

```diff
@@ -48,15 +48,15 @@
 
           <div class="info">
             <ion-card>
+              <ion-card-header>
+                <ion-card-title>{{ order.customer?.name }}</ion-card-title>
+                <!-- TODO: handle this property to display loyalty options -->
+                <ion-chip v-if="order.customer?.loyaltyOptions">
```

Comment:
Please check figma and match chip styling

Reply:
I have applied outline to this chip according to the figma design.